src/main/kotlin/com/example/iotmanagementdevice/model/MongoDevice.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.iotmanagementdevice.model
+
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.TypeAlias
+import org.springframework.data.mongodb.core.mapping.Document
+
+@TypeAlias("Device")
+@Document(collection = MongoDevice.COLLECTION_NAME)
+data class MongoDevice(
+    @Id
+    val id: ObjectId?,
+    val name: String?,
+    val description: String?,
+    val type: String?,
+    val statusType: DeviceStatusType?,
+    val userId: ObjectId?,
+) {
+    override fun toString(): String {
+        return "Device(id=$id, name=$name, description=$description, " +
+            "type=$type, statusType=$statusType, userId=$userId)"
+    }
```

Comment:
Where can we use this?
Besides, it's data class, so overriding **toString()** is bad idea

Reply:
I override it due to bidirectional relation with device and user

Reply:
due to this https://youtrack.jetbrains.com/issue/KT-16244

Reply:
but I don't see any relations in this class

Reply:
u are right, just delete it. Forgot that changed to list of object ids when migrate from sql